lib/PushController.js

```diff
@@ -4,15 +4,9 @@ var _ = require('lodash'),
     gcmPusher = require('./GCMPusher');
 
 
-var send = function (pushAssociations, androidPayload, iosPayload) {
-    var androidTokens = _(pushAssociations).where({type: 'android'}).map('token').value();
+var send = function (pushAssociations, iosPayload) {
```

Comment:
instead of removing all mention of android here and delegating down to the ios/apnPusher I would change `iosPayload` to `payload` here

and I would move the logic into this main send function about whether to call the existing apnPusher method or to use the new router to the new twilio service.  this is the right place to put it.

Reply:
Since we want to move redirection to this function, I think it would be more convenient to pass the whole `notif` object here to get user ids and to do some relevant checks.

Please, check out my change.